prow/config.yaml

```diff
@@ -1165,7 +1165,7 @@ presubmits:
     trigger: "(?m)^/test (all|pull-tekton-pipeline-build-tests),?(\\s+|$)"
     spec:
       containers:
-      - image: gcr.io/tekton-releases/dogfooding/test-runner@sha256:01bace96516d22d70c092ff9c01f9ffe7af806c7475157dcaa7cdb45d84bd68b # golang 1.18.7
```

Comment:
It would probably be worth replacing this SHA in all the jobs.

Reply:
Doh! I naively thought this would be defined in one place and used everywhere.